### PR TITLE
new ValueSet - Task.status - Fixes status to set of codes used in an AU Context

### PR DESCRIPTION
**Pull request by mjosborne1**
This PR fixes [FHIR-47081](https://jira.hl7.org/browse/FHIR-47160).

It excludes those status codes not required in the Australian eRequesting context ('draft','ready','entered-in-error')
Complete hl7au Ballot Resolution for Task.status bound ValueSet change see - [Github](https://github.com/orgs/hl7au/projects/10/views/1?pane=issue&itemId=80671787)

Changes:
TBD

### CODE REVIEW

**Comment by dt-r** [comment]
> Complete hl7au Ballot Resolution for Task.status bound ValueSet change see - [Github](https://github.com/orgs/hl7au/projects/10/views/1?pane=issue&itemId=80671787)

Please amend in the Jira ticket for this change in the PR description. Add the text "This PR fixes ...".

**Review by brettesler-ext — APPROVED** [review]
Note https://jira.hl7.org/browse/FHIR-47081 needs to incorporate this ValueSet

File: input/pagecontent/terminology.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Many value sets used in this guide are defined in the base FHIR specification, [
 |[Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|NCTS|
 |[RequestIntent](https://hl7.org/fhir/R4/valueset-request-intent.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
 |[RequestStatus](https://hl7.org/fhir/R4/valueset-request-status.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
+|[TaskStatus](http://hl7.org.au/fhir/ereq/ValueSet/task-status.html)|[AU eRequesting Task](StructureDefinition-au-erequesting-task.html)|FHIR|
 {:.grid}
```
Comment (dbojicic) [inline input/pagecontent/terminology.md:40]: Commenting in this place in the file to get an inline comment. 
Please add TaskStatus code system to the list of code systems.

File: fsh-generated/resources/ValueSet-task-status.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
```
Comment (dbojicic) [inline fsh-generated/resources/ValueSet-task-status.json:1]: Commenting here to get an inline comment. 

Please update change log - related JIRA ticket: https://jira.hl7.org/browse/FHIR-47081

File: input/pagecontent/terminology.md [hunk 3]
```diff
@@ -36,6 +36,7 @@ Many value sets used in this guide are defined in the base FHIR specification, [
 |[Reason for Request](https://healthterminologies.gov.au/fhir/ValueSet/reason-for-request-1)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|NCTS|
 |[RequestIntent](https://hl7.org/fhir/R4/valueset-request-intent.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
 |[RequestStatus](https://hl7.org/fhir/R4/valueset-request-status.html)|[AU eRequesting ServiceRequest](StructureDefinition-au-erequesting-servicerequest.html)|FHIR|
+|[TaskStatus](http://hl7.org.au/fhir/ereq/ValueSet/task-status.html)|[AU eRequesting Task](StructureDefinition-au-erequesting-task.html)|FHIR|
```
Comment (dbojicic) [inline input/pagecontent/terminology.md:39]: For the value set entry here, the value set title hyperlinked to its definition should be included: [AU eRequesting Task Status](ValueSet-task-status.html)
Published column should be: 'AU eRequesting'

File: fsh-generated/resources/ValueSet-task-status.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  "resourceType": "ValueSet",
+  "status": "draft",
+  "name": "TaskStatus",
```
Comment (dbojicic) [inline fsh-generated/resources/ValueSet-task-status.json:4]: name and title should align. On what those values should be - do we have any conventions on refining core terminology?

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see inline comments.

File: fsh-generated/resources/ValueSet-task-status.json (hunk 4 above)
Comment (mjosborne1) [reply fsh-generated/resources/ValueSet-task-status.json:4]: Updated terminology page to reflect AU eRequesting Task Status ValueSet name

File: input/pagecontent/terminology.md (hunk 3 above)
Comment (mjosborne1) [reply input/pagecontent/terminology.md]: Updated terminology page to reflect AU eRequesting Task Status ValueSet name

File: fsh-generated/resources/ValueSet-task-status.json (hunk 2 above)
Comment (mjosborne1) [reply fsh-generated/resources/ValueSet-task-status.json:1]: It appears that 47801 is already mentioned in the change log file.

File: input/pagecontent/terminology.md (hunk 1 above)
Comment (mjosborne1) [reply input/pagecontent/terminology.md:40]: Added Task Status to Code System section.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Just one more small change.

**Pull request merged**